packages/synapse-react-client/src/components/FullTextSearch.tsx

```diff
@@ -33,31 +33,31 @@ export const FullTextSearch: React.FunctionComponent<FullTextSearchProps> = ({
         `Search term must have a minimum of ${MIN_SEARCH_QUERY_LENGTH} characters`,
       )
     } else {
-      const lastQueryRequestDeepClone = getLastQueryRequest()
+      executeQueryRequest(request => {
```

Comment:
In general, make changes to pass a transforming function to `executeQueryRequest` instead of passing a (possibly stale) object.